sdk/storage/Azure.Storage.Queues/src/QueueMessageCodec.cs

```diff
@@ -41,12 +41,12 @@ public static BinaryData DecodeMessageBody(string messageText, QueueMessageEncod
 
             switch (messageEncoding)
             {
-                case QueueMessageEncoding.UTF8:
+                case QueueMessageEncoding.None:
                     return new BinaryData(messageText);
                 case QueueMessageEncoding.Base64:
                     return new BinaryData(Convert.FromBase64String(messageText));
                 default:
-                    throw new ArgumentException($"Unsupported message encoding {messageEncoding}");
+                    throw new ArgumentException($"Unsupported message encoding {messageEncoding}", nameof(messageEncoding));
```

Comment:
There's another one on line 31.

Reply:
Yeah. I noticed that too and changed in one of subsequent commits.